arch/arm/src/common/arm_internal.h

```diff
@@ -427,6 +439,18 @@ void arm_prefetchabort(uint32_t *regs);
 uint32_t *arm_syscall(uint32_t *regs);
 void arm_undefinedinsn(uint32_t *regs);
 
+/* IRQ Flag */
+
+noinstrument_function
+static inline_function void arm_set_irq_flag(bool flag)
```

Comment:
```suggestion
static inline_function void up_set_interrupt_context(bool flag)
```
get/set g_interrupt_context should be pair.

Reply:
up_interrupt_context  is get_interrupt_context